modules/sharethroughBidAdapter.js

```diff
@@ -1,123 +1,178 @@
 import { registerBidder } from '../src/adapters/bidderFactory.js';
 import * as utils from '../src/utils.js';
 import { config } from '../src/config.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { createEidsArray } from './userId/eids.js';
 
-const VERSION = '3.4.1';
+const VERSION = '4.0.0';
 const BIDDER_CODE = 'sharethrough';
-const STR_ENDPOINT = 'https://btlr.sharethrough.com/WYu2BXv1/v1';
-const DEFAULT_SIZE = [1, 1];
+const SUPPLY_ID = 'WYu2BXv1';
+
+const STR_ENDPOINT = `https://btlr.sharethrough.com/universal/v1?supply_id=${SUPPLY_ID}`;
 
 // this allows stubbing of utility function that is used internally by the sharethrough adapter
 export const sharethroughInternal = {
-  b64EncodeUnicode,
-  handleIframe,
-  isLockedInFrame,
-  getProtocol
+  getProtocol,
 };
```

Comment:
Is this still necessary? It seems there are a good amount of utility functions added in this PR that aren't using this namespace. I'm fine with leaving it as-is, simply curious what the intent is for the one function.

Reply:
@ncolletti it was originally to allow stubbing in specs - right now `getProtocol` is the only method that we need to mock in order to run unit tests, but in the past we've had more. If this isn't an issue I'd favor keeping it